wcfsetup/install/files/lib/util/StringUtil.class.php

```diff
@@ -47,7 +47,7 @@ public static function getHash($value) {
 	 * @return	string
 	 */
 	public static function getRandomID() {
-		return bin2hex(random_bytes(20));
+		return bin2hex(\random_bytes(20));
```

Comment:
If you're really making this change in order to improve the security, then you should also adjust `bin2hex()`, otherwise this can be abused too.

Reply:
Throughout the code-base?

Reply:
Nah, just at this point is sufficient, after all this is about guaranteeing a reliable source.

The performance improvement is neglible. If you're not using opcache, you don't care for performance at all and an extra "namespace lookup" is by far the least of your problems.